pydy/forces.py

```diff
@@ -0,0 +1,35 @@
+from sympy import Symbol
+
+__all__ = ['Force', 'GravitationalForce']
+
+
+class Force(object):
+    """
+    It provides an concept of force, although actual force is added by system.
```

Comment:
This sentence is not very clear. I'm not sure what it means and is it even necessary to mention system here?